PurchasesCoreSwiftTests/LocalReceiptParsing/Builders/InAppPurchaseBuilderTests.swift

```diff
@@ -36,6 +36,17 @@ class InAppPurchaseBuilderTests: XCTestCase {
         let sampleReceiptContainer = sampleInAppPurchaseContainerWithMinimalAttributes()
         expect { try self.inAppPurchaseBuilder.build(fromContainer: sampleReceiptContainer) }.notTo(throwError())
     }
+    
+    func testBuildThrowsIfUnexpectedNumberOfInternalContainers() {
+        let typeContainer = containerFactory.intContainer(int: 1)
+        let versionContainer = containerFactory.intContainer(int: 1)
+        let valueContainer = containerFactory.constructedContainer(containers: [containerFactory.stringContainer(string: "test")])
+        let unexpectedContainer = containerFactory.intContainer(int: 2)
+        
+        let inAppPurchaseContainer = containerFactory.constructedContainer(containers: [typeContainer, versionContainer, valueContainer, unexpectedContainer])
+        
+        expect { try self.inAppPurchaseBuilder.build(fromContainer: inAppPurchaseContainer) }.to(throwError())
```

Comment:
this might not be obvious if you're not used to swift, but the `expect ... throwError` doesn't catch `fatalErrors`, since they're a different _kind_ of failure.

Reply:
so this fails on develop